game_eggs/steamcmd_servers/black_mesa/egg-black-mesa.json

```diff
@@ -0,0 +1,92 @@
+            "container": "debian:buster-slim",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Game ID",
+            "description": "nmrih id",
+            "env_variable": "SRCDS_APPID",
+            "default_value": "346680",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|string|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "Game Name",
+            "description": "game name",
+            "env_variable": "SRCDS_GAME",
+            "default_value": "bms",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|string|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "Default Map",
+            "description": "default map",
+            "env_variable": "SRCDS_MAP",
+            "default_value": "dm_crossfire",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:25",
+            "field_type": "text"
+        },
+        {
+            "name": "Client port",
+            "description": "client port",
+            "env_variable": "SRCDS_CPORT",
+            "default_value": "27005",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|regex:\/^(\\w{1,20})$\/",
+            "field_type": "text"
+        },
+        {
+            "name": "Source TV",
+            "description": "source TV",
+            "env_variable": "SRCDS_STV",
+            "default_value": "27020",
+            "user_viewable": true,
+            "user_editable": false,
+            "rules": "required|string|max:20",
+            "field_type": "text"
+        },
+        {
+            "name": "Max Players",
+            "description": "The maximum amount of players allowed on your game server.",
+            "env_variable": "MAX_PLAYERS",
```

Comment:
Is this a required option? I prefer not to add these unless absolutely necessary.

Reply:
yes required (in default tutorial)